Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -646,139 +646,139 @@ Don't have a 4chan Pass?<br>
+    <string name="could_not_initialized_captcha">Could not present CAPTCHA because %1$s</string>
     <string name="fail_reason_webview_is_not_installed">WebView is not installed</string>
-    <string name="fail_reason_some_part_of_webview_not_initialized">Some part of WebViewChromium couldn\'t get initialized: (%1$s)</string>
+    <string name="fail_reason_some_part_of_webview_not_initialized">Could not start some part of WebViewChromium because %1$s</string>
 
-    <string name="setting_enable_image_link_loading">Enable image link loading</string>
-    <string name="setting_enable_image_link_loading_description">Adds any JPG, PNG, GIF, WEBM, MP4, PDF, BMP, WEBP, MP3, SWF, M4A, OGG, or FLAC files linked in posts as internally viewable post attachments with a special spoiler image.</string>
+    <string name="setting_enable_image_link_loading">Image link loading</string>
+    <string name="setting_enable_image_link_loading_description">Adds any JPG, PNG, GIF, WebM, MP4, PDF, BMP, WebP, MP3, SWF, M4A, OGG, or FLAC files linked in posts as internally viewable post attachments with a special spoiler image.</string>
     <string name="add_new_zone">Add new zone</string>
-    <string name="setting_exclusion_zones_left_zone_portrait">Left zone (Portrait orientation)</string>
-    <string name="setting_exclusion_zones_right_zone_portrait">Right zone (Portrait orientation)</string>
-    <string name="setting_exclusion_zones_left_zone_landscape">Left zone (Landscape orientation)</string>
-    <string name="setting_exclusion_zones_right_zone_landscape">Right zone (Landscape orientation)</string>
+    <string name="setting_exclusion_zones_left_zone_portrait">← zone (▯)</string>
+    <string name="setting_exclusion_zones_right_zone_portrait">→ zone (▯)</string>
+    <string name="setting_exclusion_zones_left_zone_landscape">← zone (▭)</string>
+    <string name="setting_exclusion_zones_right_zone_landscape">→ zone (▭)</string>
     <string name="setting_exclusion_zones_reset_zones">Reset exclusion zones</string>
-    <string name="setting_exclusion_zones_reset_zones_description">All exclusion zones will be deleted; the app will be restarted</string>
-    <string name="setting_exclusion_zones_editor">Android 10 gesture exclusion zones</string>
-    <string name="setting_exclusion_zones_editor_description">Adjust zones where Android 10 gestures will be disabled. \n\nYou will have to rotate your phone/tablet to set zones for different orientations. \nYou will have to restart the app for the changes to be applied!</string>
+    <string name="setting_exclusion_zones_reset_zones_description">All exclusion zones will be deleted and the app restarted.</string>
+    <string name="setting_exclusion_zones_editor">Android 10 gesture-exclusion zones</string>
+    <string name="setting_exclusion_zones_editor_description">Adjust zones to turn off Android 10 gestures. \n\nRotate your device to set zones for different orientations. \nRestart the app to use changes.</string>
```

Comment:
The ``where`` is important in the original. Otherwise we need to imply that gestures turn off only in those zones. And not that using those zones could turn them off completely.

Reply:
```suggestion
    <string name="setting_exclusion_zones_editor_description">Adjust zones where Android 10 gestures are off. \n\nRotate your device to set zones for different orientations. \nRestart the app to use changes.</string>
```